src/components/__test__/Subscribe.test.js

```diff
@@ -0,0 +1,8 @@
+import renderer from "react-test-renderer";
+
+import Subscribe from "./../Subscribe";
```

Comment:
You can make is simpler by using code like this
`import Subscribe from "../Subscribe";`